test/suite/test_bug025.py

```diff
@@ -55,31 +55,31 @@ def test_bug025(self):
         pos = os.path.getsize(filename) - 1024
         os.remove(filename)
 
-        # We get an error message, but the open succeeds.
-        with self.expectedStderrPattern('.*No such file or directory'):
+        # We will get error output, but not always in the same API calls from run to run,
+        # in particular the open connection doesn't always report the missing file, as
+        # index files are usually lazily loaded. As long as the missing file is reported
+        # at least once in the following code, it's good.  We're doing a pretty bad thing
+        # by removing the index file, whether we get two messages or three is not important.
```

Comment:
I think the last sentence is unnecessary and a bit confusing about number of messages. The previous sentence sums it up nicely. I suggest removing the last comment sentence.